pkg/ansible/watches/watches.go

```diff
@@ -304,6 +342,8 @@ func Load(path string, maxWorkers, ansibleVerbosity int) ([]Watch, error) {
 			log.Error(err, fmt.Sprintf("Watch with GVK %v failed validation", watch.GroupVersionKind.String()))
 			return nil, err
 		}
+
+		fmt.Printf("%+v", watch)
```

Comment:
revise this?

Reply:
Yeah looks like a leftover debug print, should be removed